src/AdminService.Business/Commands/FindAdminCommand.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using LT.DigitalOffice.AdminService.Business.Commands.Interfaces;
+using LT.DigitalOffice.AdminService.Data.Interfaces;
+using LT.DigitalOffice.AdminService.Mappers.Interfaces;
+using LT.DigitalOffice.AdminService.Models.Db;
+using LT.DigitalOffice.AdminService.Models.Dto.Models;
+using LT.DigitalOffice.AdminService.Models.Dto.Requests.Filtres;
+using LT.DigitalOffice.Kernel.Enums;
+using LT.DigitalOffice.Kernel.FluentValidationExtensions;
+using LT.DigitalOffice.Kernel.Helpers;
+using LT.DigitalOffice.Kernel.Helpers.Interfaces;
+using LT.DigitalOffice.Kernel.Responses;
+using LT.DigitalOffice.Kernel.Validators.Interfaces;
+using Microsoft.AspNetCore.Http;
+
+namespace LT.DigitalOffice.AdminService.Business.Commands
+{
+  public class FindAdminCommand : IFindAdminCommand
+  {
+    private readonly IBaseFindFilterValidator _baseFindValidator;
+    private readonly IServiceConfigurationRepository _configrepository;
+    private readonly IConfigurationServicesInfoMapper _configmapper;
+    private readonly IHttpContextAccessor _httpContextAccessor;
+    private readonly IResponseCreater _responseCreator;
+
+    public FindAdminCommand(
+      IBaseFindFilterValidator baseFindValidator,
+      IServiceConfigurationRepository configrepository,
+      IConfigurationServicesInfoMapper configmapper, 
+      IHttpContextAccessor httpContextAccessor,
+      IResponseCreater responseCreator)
+    {
+      _baseFindValidator = baseFindValidator;
+      _configrepository = configrepository;
+      _configmapper = configmapper;
+      _httpContextAccessor = httpContextAccessor;
+      _responseCreator = responseCreator;
+    }
+
+    public async Task<FindResultResponse<ConfigurationServicesInfo>> ExecuteAsync(FindAdminFilter filter)
+    {
+      if(!_baseFindValidator.ValidateCustom(filter, out List<string> errors))
+      {
+        return _responseCreator.CreateFailureFindResponse<ConfigurationServicesInfo>(HttpStatusCode.BadRequest, errors);
+      };     
+
+      FindResultResponse<ConfigurationServicesInfo> response = new();
+      response.Body = new();
+
+      (List<DbServiceConfiguration> dbConfig, int totalCount) findServicesResponse =

```

Comment:
you can do not crete  findServicesResponse vdriable: (List<DbServiceConfiguration> dbConfig, int totalCount) = ...